packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -354,6 +355,12 @@ export interface IGCRuntimeOptions {
 	 */
 	sessionExpiryTimeoutMs?: number;
 
+	/**
+	 * Delay between when Tombstone should run and when the object should be deleted.
+	 * If not present, a default (non-zero) valua will be used.
```

Comment:
Do consumers need to know what this default value is? Or is this intended to be policy we can change at any time?

Reply:
I intentionally ommitted it, since it's not defined here and I want to avoid drift.  I don't think it's important what the default value is, if someone is engaging with this feature they can look at the code to find out, but they probably will want to set it proactively.